package.json

```diff
@@ -1,19 +1,40 @@
 {
   "name": "timberlake",
-  "version": "1.0.0",
+  "version": "0.0.1",
```

Comment:
Ups, I missed the fact that package.json was already there. Reverted the unnecessary changes.